src/hotspot/cpu/x86/x86.ad

```diff
@@ -3232,73 +3232,22 @@ instruct negD_reg_reg(vlRegD dst, vlRegD src) %{
   ins_pipe(pipe_slow);
 %}
 
-instruct sqrtF_reg(regF dst, regF src) %{
```

Comment:
Would be helpful to have a comment describing why there are only reg-to-reg variants kept for `SqrtF`/`SqrtD`.

Reply:
Done, added comments for the sqrt rules.